src/diffusers/models/embeddings.py

```diff
@@ -683,3 +718,110 @@ def forward(
             objs = torch.cat([objs_text, objs_image], dim=1)
 
         return objs
+
+
+class SizeEmbedder(nn.Module):
```

Comment:
Can we find a better name here? `SizeEmbedder` is a bit weird IMO

Reply:
Actually I don't think we need this class at all - can't we just re-use https://github.com/huggingface/diffusers/blob/080081bdeda61a4723a61594f9f1c12909d695b7/src/diffusers/models/embeddings.py#L159

and then handle all the timestep creation inside: CombinedTimestepSizeEmbeddings?